lib/matplotlib/axes/_base.py

```diff
@@ -113,9 +116,14 @@ def _process_plot_format(fmt):
                 raise ValueError(
                     'Illegal format string "%s"; two color symbols' % fmt)
             color = c
+        elif c == 'C' and i < len(chars) - 1:
+            color_cycle_number = int(chars[i + 1])
```

Comment:
Limited to 10 I guess, but what if you wanted to use something like Vega20 (not that we have that as a cycle at the moment.)